client/state/sync/constants.ts

```diff
@@ -13,6 +13,4 @@ export enum SiteSyncStatusProgress {
 	RESTORE = 0.6,
 	COMPLETED = 1,
 	FAILED = 0,
-	ALLOW_RETRY = 0.1,
```

Comment:
This change is related to the `should set isSyncingInProgress to false if status become allow_retry` test, where we expect the progress to be set to `1` (the same as for the completed state):

https://github.com/Automattic/wp-calypso/blob/d70973fcf281fd7b52cb32bfd2967f2d933943dd/client/state/sync/test/reducer.js#L101-L122

I am curious to hear what you think.

Reply:
I don't fully understand why the ALLOW_RETRY should mark the progress to 1 and keep isSyncingInProgress to true. My first thought would be setting the progress value to 0.

Reply:
Thank you for the review, Antonio.

When the sync status becomes `ALLOW_RETRY`, we expect `isSyncingInProgress` to go to `false` in that test. That part seems good to me.

But it is not clear to me why we expect the `progress` to go to `1`. Regardless, I have tested changing the state to `ALLOW_RETRY` by following the steps on the bottom of this diff: D131105-code. That does not seem to have any effect though.

Hey @wojtekn 👋🏼 By any chance, do you remember why we are expecting the staging site sync `progress` to be set to `1` when the status is `ALLOW_RETRY`? I see the related test was introduced in https://github.com/Automattic/wp-calypso/pull/84955.

Reply:
Thank you, Wojtek! I have updated the test and see you have already approved the PR in the meantime.

Thank you both for your review. Going to merge and deploy the PR now.